docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -62,6 +64,18 @@ Perhaps. GitHub permissions can quickly get complicated, as it involves two leve
 
 First of all, you can only configure already-installed GitHub Apps for organizations you are not an admin in if you go through the app installation process again. Be aware that in this scenario, you can only add repos that you have admin access to, not just maintainer access.
 
+There are two scenarios to be aware of when installing the Dockstore GitHub App as a repository admin who is not an organization admin.
+
+1. An organization admin configured the Dockstore GitHub App to be installed on all repositories.
```

Comment:
Would reword this to "Another person, who is an organization admin,"

Reply:
You may also want to reword "to be installed on all repositories" to something like "to have universal access to all repositories in the organization" because installing manually on all repos works differently, I think.

Reply:
> You may also want to reword "to be installed on all repositories" to something like "to have universal access to all repositories in the organization" because installing manually on all repos works differently, I think.

I reworded it to "to be installed on all current and future repositories" to match the phrasing in Denis' [screenshot](https://github.com/dockstore/dockstore-documentation/pull/225#discussion_r973069384)